profiles: Add /usr/libexec as a path to the libvirt leaseshelper script

openSUSE recently joined most distros in defining libexecdir as /usr/libexec.
The SUSE libvirt package, which for a long time has set libexecdir to
/usr/lib64/libvirt, needs to adopt. Jira SLE-14253 requests libvirt to use
/usr/libexec. libvirt 6.7.0 will be hitting Factory soon with libexecdir
set to /usr/libexec. Add it as a path for the libvirt_leaseshelper script.

Signed-off-by: Jim Fehlig <jfehlig@suse.com>
Index: apparmor-2.13.4/profiles/apparmor.d/usr.sbin.dnsmasq
===================================================================
--- apparmor-2.13.4.orig/profiles/apparmor.d/usr.sbin.dnsmasq
+++ apparmor-2.13.4/profiles/apparmor.d/usr.sbin.dnsmasq
@@ -88,7 +88,7 @@ profile /usr/sbin/dnsmasq /usr/{bin,sbin
   /{,var/}run/libvirt/network/*.pid rw,
 
   # libvirt lease helper
-  /usr/lib{,64}/libvirt/libvirt_leaseshelper Cx -> libvirt_leaseshelper,
+  /usr/{lib/libvirt,lib64/libvirt,libexec}/libvirt_leaseshelper Cx -> libvirt_leaseshelper,
 
   # lxc-net pid and lease files
   /{,var/}run/lxc/dnsmasq.pid    rw,
@@ -115,7 +115,7 @@ profile /usr/sbin/dnsmasq /usr/{bin,sbin
 
     /etc/libnl-3/classid r,
 
-    /usr/lib{,64}/libvirt/libvirt_leaseshelper m,
+    /usr/{lib/libvirt,lib64/libvirt,libexec}/libvirt_leaseshelper m,
 
     owner @{PROC}/@{pid}/net/psched r,
     owner @{PROC}/@{pid}/status r,
